Tasks/GradleV2/Tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Agent.Tasks.Gradle",
+  "version": "2.139.1",
+  "author": "Microsoft Corporation",
+  "description": "Build with Gradle",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vso-agent-tasks/issues"
+  },
+  "dependencies": {
```

Comment:
I think this should be `devDependencies` since they aren't needed at runtime for the task.  @stephenmichaelf , what are your thoughts?